### ISSUE TITLE
Unhandled exception: [Enter a title]

### ISSUE THREAD

**Issue by dazric** [remyroy/CDDA-Game-Launcher#33]
* Description: [Enter what you did and what happened]
* Version: 0.8.1
* Type: `<class 'FileNotFoundError'>`
* Value: [Errno 2] No such file or directory: 'C:\\Users\\Peter\\Downloads\\cdda\\data\\mods\\Only_Wildlife\\modinfo.json'
* Traceback:
```
  File "cddagl\ui.py", line 558, in showEvent
  File "cddagl\ui.py", line 733, in game_directory_changed
  File "cddagl\ui.py", line 680, in update_mods
  File "cddagl\ui.py", line 4892, in game_dir_changed
  File "cddagl\ui.py", line 4766, in config_info

```

**Comment by remyroy** [comment]
This is a weird one. Before trying to open that file, we make sure it exists. I guess we'll add some more exception handling in there.

**Comment by remyroy** [comment]
This should be fixed with 910dbe9c9b81367937fb526e664b106a02d2573d
